app/views/public/api/v1/portals/articles/_article_header.html.erb

```diff
@@ -1,17 +1,31 @@
-<h1 class="text-3xl font-semibold md:tracking-normal leading-snug md:text-4xl text-slate-900 dark:text-white">
+<div class="flex items-center flex-row mb-6 gap-px">
+  <a
+    class="text-slate-500 dark:text-slate-200 text-sm gap-1 hover:cursor-pointer <%= @is_plain_layout_enabled && 'hover:underline' %> leading-8 font-semibold"
+    href="/hc/<%= @portal.slug %>/<%= @article.category&.locale %><%= @theme.present? && @theme != 'system' ? '?theme='+@theme : ''  %>"
+  >
+    <%= I18n.t('public_portal.common.home') %>
+  </a>
+  <span class="w-4 h-4 [&>svg]:w-3 [&>svg]:h-3 flex items-center justify-center text-xs text-slate-500 dark:text-slate-300"><%= render partial: 'icons/chevron-right' %></span>
+  <% if @article.category %>
+    <a class="text-slate-500 dark:text-slate-200 text-sm gap-1 whitespace-nowrap hover:cursor-pointer <%= @is_plain_layout_enabled && 'hover:underline' %> leading-8 font-semibold" href="/hc/<%= @portal.slug %>/<%= @article.category.locale %>/categories/<%= @article.category.slug %><%= @theme.present? && @theme != 'system' ? '?theme='+@theme : ''  %>">
```

Comment:
@sojan-official Can we do anything for links like this? We are repeating these at many places

Reply:
@nithindavid It's handled in the last PR https://github.com/chatwoot/chatwoot/pull/8230/files#diff-507eaef3ab83df007fe2b82a5364439aebd3fa8fa2ef5234eaf5bcd050754447R47 . Now we have three helpers for opening links to the home page, category page, and article page.